### PR TITLE
Adding bigger icons and updating one used for PWA

### PR DESCRIPTION
**Pull request by vinzv**
Tested with Firefox Mobile and Chrome on Android.
Fixing #9337

### CODE REVIEW

**Comment by annando** [comment]
You added the 512 pixel version as well - but didn't link it. Is this intentional?

File: src/Module/Manifest.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public static function rawContent(array $parameters = [])
 			'icons'       => [
 				[
```
Comment (annando) [inline src/Module/Manifest.php:45]: I guess you should add the support for the 512pixel icons as well.

**Comment by annando** [comment]
Please change this occasion here to the larger icon as well: https://github.com/friendica/friendica/blob/develop/src/App/Page.php#L236

**Comment by vinzv** [comment]
Yes, as it's only recommended but not necessary for validity. Should I add it to `icons` as well?

File: src/Module/Manifest.php (hunk 1 above)
Comment (vinzv) [reply src/Module/Manifest.php:45]: My thought was to not add more than what's required. But you're right, adding the recommended one as well should do no harm.

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Thank you for your work!

**Pull request merged**